### PR TITLE
Feat/add footer

### PR DESCRIPTION
**Pull request by D-Kondratiev**

### CODE REVIEW

File: src/components/Footer/Footer.scss [hunk 1]
```diff
@@ -0,0 +1,117 @@
+@import '/src/prepare-styles/colors.scss';
+@import '/src/prepare-styles/mixins.scss'; 
+@import '/src/prepare-styles/usable-fonts.scss'; 
+
+$tablet-min-width: 640px;
```
Comment (yanamomot) [inline src/components/Footer/Footer.scss]: Please, move this constants to new created separate file(prepare-styles/constant.scss) for example))

File: src/components/Footer/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+$desktop-min-width: 1200px;
+
+
+
+@mixin content-padding-inline {
```
Comment (yanamomot) [inline src/components/Footer/Footer.scss]: Move to mixins file too

File: src/components/Footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  }
+}
+
+.container {
+  font-family: 'Mont-Regular'; 
+  @include content-padding-inline;
+}
+
+.footer {
+  position: relative;
```
Comment (yanamomot) [inline src/components/Footer/Footer.scss]: Honestly, I dont understand why you use position relative(if i remember right, its default)

File: src/components/Footer/Footer.scss [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+  &__label {
+    font-family: 'Mont-SemiBold'; 
+    font-weight: 700;
+    font-size: 12px;
+    color: $graySecondary;
+    margin-right: 16px;
+  }
+}
+
```
Comment (yanamomot) [inline src/components/Footer/Footer.scss:99]: As for me, other code look like good. Please, fix that minor comments, i hope that would be easy for You)

**Review by yanamomot — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.scss [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+.footer {
+  position: relative;
+  display: block;
+  text-align: left;
+  padding-block: 32px;
+  border-top: 1px solid $grayElements;
+
+  @include on-tablet {
+    flex-wrap: wrap;
```
Comment (yanamomot) [inline src/components/Footer/Footer.scss:17]: Also, i dont remember can we use flex-wrap without flex(on that case display block)

**Comment by kinqbert** [comment]
i have nothing more to say. fix Yana's comments and will be approved by me 🤝

**Review by yanamomot — APPROVED** [review]
Great! Thank`s🤗

**Review by kinqbert — APPROVED** [review]

**Pull request merged**